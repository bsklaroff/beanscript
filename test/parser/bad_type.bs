
type Bad = | A

print(0)
