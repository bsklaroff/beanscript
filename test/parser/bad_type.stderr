Parse error, line 2:11 of user input
type Bad = | A
          ^
